genref/output/md/kubeadm-config.v1beta2.md

```diff
@@ -156,90 +153,90 @@ configuration types to be used during a `kubeadm init` run.
 apiVersion: kubeadm.k8s.io/v1beta2
 kind: InitConfiguration
 bootstrapTokens:
-- token: "9a08jv.c0izixklcxtmnze7"
-  description: "kubeadm bootstrap token"
-  ttl: "24h"
-- token: "783bde.3f89s0fje9f38fhf"
-  description: "another bootstrap token"
-  usages:
-  - authentication
-  - signing
-  groups:
-  - system:bootstrappers:kubeadm:default-node-token
+  - token: "9a08jv.c0izixklcxtmnze7"
+    description: "kubeadm bootstrap token"
+    ttl: "24h"
+  - token: "783bde.3f89s0fje9f38fhf"
+    description: "another bootstrap token"
+    usages:
+      - authentication
+      - signing
+    groups:
+      - system:bootstrappers:kubeadm:default-node-token
 nodeRegistration:
-  name: "ec2-10-100-0-1"
-  criSocket: "/var/run/dockershim.sock"
-  taints:
-  - key: "kubeadmNode"
-    value: "master"
-    effect: "NoSchedule"
-  kubeletExtraArgs:
-    cgroup-driver: "cgroupfs"
-  ignorePreflightErrors:
-  - IsPrivilegedUser
+	 name: "ec2-10-100-0-1"
```

Comment:
nit: tabs or spaces?

Reply:
Will revisit this. These tabs come from the source code too.